### PR TITLE
block_hotplug: add QHPDrive support

### PR DESCRIPTION
**Pull request by navinethomas**
These changes are similar to the case in pci_hotplug.py

### CODE REVIEW

File: qemu/tests/block_hotplug.py [hunk 1]
```diff
@@ -121,8 +142,13 @@ def unplug_device(vm, get_disk_cmd, device):
                         test.fail(err)
                     else:
                         controller_list.append(controller)
+
```
Comment (ldoktor) [inline qemu/tests/block_hotplug.py]: and those tailing white characters (remove the spaces)

File: qemu/tests/block_hotplug.py [hunk 2]
```diff
@@ -121,8 +142,13 @@ def unplug_device(vm, get_disk_cmd, device):
                         test.fail(err)
                     else:
                         controller_list.append(controller)
+                
+                if drive_cmd_type == "drive_add":
+                    drive = qdevices.QHPDrive("block%d" % num)
+                    drive.set_param("if","none")
```
Comment (ldoktor) [inline qemu/tests/block_hotplug.py]: and here you miss a white space after `,`, therefor it should be `drive.set_param("if", "none")`.

**Comment by aliang123** [comment]
https://github.com/autotest/tp-qemu/pull/1396 has resolved this issue, please help to review, thank you.

**Comment by ldoktor** [comment]
OK, let's move the discussion there and close this one...

**Pull request closed without merging**